docs/onboarding/06-e2e-sample.md

```diff
@@ -0,0 +1,35 @@
+# Chapter 6: Deploy and run end-to-end sample
+
+The following docs will introduce an end-to-end solution where an Azure Digital Twins instance is connected to other Azure services for management of devices and data.
```

Comment:
Aside from the tutorial, is there any other content you have encountered that describes how e2e solutions are typically architected for ADT? For example, are there any public customer use case videos that describe their architecture? Is there any content beyond the docs.microsoft.com that might provides some different perspectives?

This part of the doc should include links to any reading or videos that teach the concepts that will need to be applied in the challenge.

Reply:
I struggled with that one. I ended up adding some links to more complete examples for ingesting from IoT hub, and for managing endpoints and routes. Any other ideas on appropriate content would be welcome. @TechnicallyWilliams  @charleszipp